main.tf

```diff
@@ -1,6 +1,9 @@
 module "repository" {
-  for_each                  = { for repo in var.repository : repo.name => repo }
-  source                    = "./modules/repository"
+  for_each = { for repo in var.repository : repo.name => repo }
+
+  source  = "damacus/github-repository/module"
+  version = "0.0.3"
```

Comment:
Can you please elaborate on why this change is needed?

Reply:
Having it as an external module allows us to version pin, I'll fix the module separately. 

I'm also going to do the same thing over at test-kitchen some time soon

Reply:
But where does this source actually exist? I'm still a little confused